docs/user_guide/emummc/partitioning_sd.md

```diff
@@ -8,14 +8,13 @@
 
 -----
 
-### What you need (Windows)
+## Windows Instructions
```

Comment:
This is inconsistent with the linux instructions. Revert this please.

Reply:
https://github.com/nh-server/switch-guide/pull/40/files#diff-5e1f5782c273e8660496c5d92f686da4R63 - it's consistent?

Reply:
caught the error: it didn't show up in the built version: line 62 needs a delete

Reply:
Meme and I fixed it.